app/orm-adapters/node-orm2.js

```diff
@@ -7,7 +7,7 @@ import snakeCase from 'lodash/snakeCase';
 import upperFirst from 'lodash/upperFirst';
 
 const typeMap = {
-  text: String,
+  string: String,
```

Comment:
This type map is meant to correspond to [node-orm2's data types](https://github.com/dresende/node-orm2/wiki/Model%20Properties#types), which uses 'text' for any string value.